src/commcare_cloud/ansible/roles/redis/tasks/setup_cluster.yml

```diff
@@ -3,13 +3,5 @@
   command: 'gem install redis -v 3.3.3' # not the latest, because https://github.com/antirez/redis/issues/4272
   become: yes
 
-- name: Add cluster config
```

Comment:
I don't think this should be removed. Rather we should change the condition in `redis/tasks/main.yml`:
```
- include: setup_cluster.yml
  when: is_redis_cluster|bool and inventory_hostname in groups['redis_cluster_master']
```